themes/digital.gov/layouts/partials/core/head.html

```diff
@@ -121,6 +121,8 @@
   <meta property="og:site_name" content="Digital.gov" />
   {{ if $featImgURL -}}
     <meta property="og:image" content="{{- $featImgURL -}}" />
+    <meta property="og:image:alt" content=" " />
+    <meta property="og:locale" content="en_US" />
```

Comment:
We should move out of the conditional block above it in case `$featImgURL` is not set.

Reply:
good call... i moved out of conditional